https://bugs.gentoo.org/981698
https://github.com/containers/bubblewrap/commit/bb3ff51ec60b40ebf0f51b33521967213f5d857e

From bb3ff51ec60b40ebf0f51b33521967213f5d857e Mon Sep 17 00:00:00 2001
From: Duncan Overbruck <mail@duncano.de>
Date: Sun, 30 Aug 2026 04:08:53 +0200
Subject: [PATCH] Add limits.h include for PATH_MAX

This is required for musl libc, where limits.h is not implicitly included
by the other includes.

Signed-off-by: Duncan Overbruck <mail@duncano.de>
---
 bubblewrap.c  | 1 +
 safe_openat.c | 1 +
 2 files changed, 2 insertions(+)

diff --git a/bubblewrap.c b/bubblewrap.c
index c35e7c58..62227014 100644
--- a/bubblewrap.c
+++ b/bubblewrap.c
@@ -21,6 +21,7 @@
 
 #include <poll.h>
 #include <sched.h>
+#include <limits.h>
 #include <pwd.h>
 #include <grp.h>
 #include <ctype.h>
diff --git a/safe_openat.c b/safe_openat.c
index a4166e6d..634f0861 100644
--- a/safe_openat.c
+++ b/safe_openat.c
@@ -21,6 +21,7 @@
 
 #include "utils.h"
 #include <sys/syscall.h>
+#include <limits.h>
 #include <unistd.h>
 #include <errno.h>
 #include <stdint.h>
